feature/interface/singleton/otg_tests/singleton_test/singleton_test.go

```diff
@@ -246,6 +272,25 @@ func (tc *testCase) verifyATE(t *testing.T) {
 	})
 }
 
+// Waits for an ARP entry to be present on ateSrc
```

Comment:
This should be two separate functions, one for IPv4 and one for IPv6. See the test case definition below. For brevity, they should just take `otg` as the argument.

Reply:
Done